ci-operator/step-registry/rosa/sts/account-roles/delete/rosa-sts-account-roles-delete-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: rosa-sts-account-roles-delete
+  from_image:
+    namespace: ci
+    name: rosa-aws-cli
+    tag: latest
+  grace_period: 10m
+  commands: rosa-sts-account-roles-delete-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 100Mi
+  env:
+  - name: ACCOUNT_ROLES_PREFIX
```

Comment:
For easier management, is it possible to use cluster name as prefix?

Reply:
We should give the user the entry to set the prefix by their preference.

Reply:
The role name should match what they used when they created the account roles. I think it might be easier to just leave the default for now to ensure that there is no mismatch.

Reply:
@vkareh The AWS account `openshift-qe` we used is a shared account, it is used in sd QE testing, ocp QE testing, or even other teams. The default prefix may be used by others, we'd better use an individual prefix to avoid conflict.

Reply:
That makes sense. In our team we use our usernames to separate the roles in our shared AWS account, so I have a set of roles with `vkareh` prefix that won't collide with my colleagues.

Reply:
current changes looks good to me.